.circleci/config.yml

```diff
@@ -42,7 +43,6 @@ commands:
       - run:
           name: Core Python Checks
           command: |
-            . venv/bin/activate
             flake8 syncserver/src/tokenserver
```

Comment:
I think we should not rely on any virtual environment and instead install packages to the system level for whatever dockerfiles we generate. Virtual environments are there to prevent multiple applications from stepping on each other. Docker images encapsulate that by giving applications a sandbox that they can run in. Therefore python's virtualenv concept is redundant in Docker images.